### PR TITLE
Flex: Stacks API Support.

### PR DESCRIPTION
**Pull request by khkh-ms**
### Issue describing the changes in this PR

resolves #issue_for_this_pr

### Pull request checklist

* [ ] My changes **do not** require documentation changes
  * [ ] Otherwise: Documentation issue linked to PR
* [ ] My changes **do not** need to be backported to a previous version
  * [ ] Otherwise: Backport tracked by issue/PR #issue_or_pr
* [ ] I have added all required tests (Unit tests, E2E tests)

### CODE REVIEW

File: src/Azure.Functions.Cli/Actions/AzureActions/PublishFunctionAppAction.cs [hunk 1]
```diff
@@ -16,7 +16,9 @@
 using Azure.Functions.Cli.Interfaces;
 using Azure.Functions.Cli.StacksApi;
 using Colors.Net;
+using Dynamitey;
```
Comment (kamperiadis) [inline src/Azure.Functions.Cli/Actions/AzureActions/PublishFunctionAppAction.cs:19]: Why do we need this?

File: src/Azure.Functions.Cli/Helpers/AzureHelper.cs [hunk 2]
```diff
@@ -722,5 +722,27 @@ public static async Task<FunctionsStacks> GetFunctionsStacks(string accessToken,
...
             }
         }
+
+        public static async Task<FlexFunctionsStacks> GetFlexFunctionsStacks(string accessToken, string managementURL, string runtime, string region)
+        {
+            // API only supports dotnet as runtime. The dotnet-isolated is part of the dotnet for this API.
+            if (runtime.Equals("dotnet-isolated", StringComparison.OrdinalIgnoreCase))
+            {
+                runtime = "dotnet";
+            }
```
Comment (kamperiadis) [inline src/Azure.Functions.Cli/Helpers/AzureHelper.cs:732]: Why are we changing this from dotnet-isolated to dotnet? The new stacks API only accepts dotnet as the runtime even though we technically only support dotnet-isolated for flex?

File: src/Azure.Functions.Cli/Actions/AzureActions/PublishFunctionAppAction.cs (hunk 1 above)
Comment (khkh-ms) [reply src/Azure.Functions.Cli/Actions/AzureActions/PublishFunctionAppAction.cs:19]: Probably auto added. Let me try to remove it.

File: src/Azure.Functions.Cli/Helpers/AzureHelper.cs (hunk 2 above)
Comment (khkh-ms) [reply src/Azure.Functions.Cli/Helpers/AzureHelper.cs:732]: Stacks API only has donnet stacks. donnet-isolated stacks are part of the "dotnet" stacks.

**Review by kamperiadis — APPROVED** [review]
LGTM

**Pull request merged**